### DISCUSSION TITLE
tabbar的具体用法

### DISCUSSION THREAD

**Discussion by kazetyan** [Moonofweisheng/wot-design-uni#83] [Q&A] [answered]
rt,是需要在每个tab的页面都添加tabbar组件,在@change里用uni.navigateTo来跳转吗?
还是说可以像vueroute那样,只在一个页面引入tabbar,然后通过路由组件来跳转

**Comment by Moonofweisheng** [comment, marked as answer]
在小程序和APP中没有`<router-view>`的，所以是需要在每个tabbar页面添加tabbar组件的

**Reply by kazetyan** [reply]
如果是每个tabbar页面都添加,会不会出现首次跳转加载慢的问题.小程序上,如果不需要特别的样式,是不是用原生的tabbar会好一些

**Comment by Moonofweisheng** [comment]
> 如果是每个tabbar页面都添加,会不会出现首次跳转加载慢的问题.小程序上,如果不需要特别的样式,是不是用原生的tabbar会好一些

tabbar，可能会出现闪烁，因为毕竟是切换页面了。如果不是需要特殊的交互和样式，推荐使用原生tabbar。

**Reply by kazetyan** [reply]
好的,谢谢

**Comment by Moonofweisheng** [comment]
> 好的,谢谢
